app/src/main/java/com/jerboa/db/AppDBMigrations.kt

```diff
@@ -469,6 +470,81 @@ val MIGRATION_30_31 =
         }
     }
 
+val MIGRATION_31_32 =
+    object : Migration(31, 32) {
+        override fun migrate(db: SupportSQLiteDatabase) {
+            // Fix wrong ordering and defaults in previous migration
+
+            db.execSQL(
+                """
+                   CREATE TABLE IF NOT EXISTS AppSettingsBackup (
+                      `id` INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL,
+                      `font_size` INTEGER NOT NULL DEFAULT ${DEFAULT_FONT_SIZE},
```

Comment:
You can't do something like this, the migrations need to stay exactly the same.

If we make migration version 33, say we change a default.

We change the constant and now we have also changed migration of 32.

Room will rightly throw a error that the checksum of migration 32 checksum mismatches. Because that means that some users their db schema 32 is different. That can be very problematic

Reply:
Alright, can change them.